modules/gapi/test/gapi_gpu_test.cpp

```diff
@@ -202,6 +203,85 @@ TEST(GPU, Symm7x7_test)
         EXPECT_EQ(out_mat_gapi.size(), sz);
     }
 }
+
+// MB it deserves own test file
```

Comment:
I think it is OK to place it here as it's a test of using MediaFrame/GFrame in OCLBackend